tests/integration/test_integration.py

```diff
@@ -281,7 +286,7 @@ def process_records(records, shard_id):
             sqs.send_message(QueueUrl=sqs_queue_info['QueueUrl'], MessageBody=str(i))
 
         LOGGER.info('Waiting some time before finishing test.')
-        time.sleep(2)
+        time.sleep(5)
```

Comment:
nitpick: Out of curiosity - is this increased sleep time required, even though we're using `stream.wait_for()` above?